### ISSUE TITLE
Building resonance audio SDK on ARM boards like Raspberry Pi or Jetson

### ISSUE THREAD

**Issue by Tetsujinfr** [resonance-audio/resonance-audio#41]
hi

is it possible to build the resonance audio SDK on ARM8 CPU boards like the Raspberry Pi or the Jetson Nano/NX, running raspbian/ubuntu respectively?

I am trying to build the SDK but I got some errors, related to SSE instruction set which do not exist on ARM (Ibuilieve the equiv is Neon instructions for ARM cpus):
```
[  2%] Building C object resonance_audio/CMakeFiles/PffftObj.dir/__/third_party/pffft/fftpack.c.o
c++: error: unrecognized command line option ‘-msse’; did you mean ‘-fdse’?
c++: error: unrecognized command line option ‘-msse2’
c++: error: unrecognized command line option ‘-msse3’

```

After cloning the main repo and dependencies repo, here is the command I entered for the build:
`./resonance-audio/build.sh -t=RESONANCE_AUDIO_TESTS -t=RESONANCE_AUDIO_API -t=GEOMETRICAL_ACOUSTICS_TESTS -t=WWISE_SOUND_ENGINE_PLUGIN`

thanks for your help.

**Comment by tristanjl** [comment]
I don't believe the platform support work has been done. You will probably have to dig into the cmake files in order to add support for non-android/ios arm builds.

Since android/iOS support is already there, I suspect you won't have to add any arm specific changes to get it to work.
